trezorlib/tests/unit_tests/test_tx_api.py

```diff
@@ -21,6 +21,7 @@
 
 TxApiBitcoin = coins.tx_api['Bitcoin']
 TxApiTestnet = tx_api.TxApiInsight("insight_testnet")
+TxApiZencash = coins.tx_api['Zencash']
```

Comment:
is there a reason to test the tx api here? if you want to have comprehensive test, you should add tests for your `get_block` method (but see other comments)